runtime-api/src/main/scala/io/coral/actors/transform/StatsActor.scala

```diff
@@ -1,22 +1,18 @@
 package io.coral.actors.transform
 
-// akka
 import akka.actor.{ActorLogging, Props}
-
-//json goodness
-import org.json4s._
+import io.coral.actors.CoralActor
+import io.coral.lib.SummaryStatistics
 import org.json4s.JsonDSL._
+import org.json4s._
 import org.json4s.jackson.JsonMethods.render
 
-// coral
-import io.coral.actors.CoralActor
-
 object StatsActor {
+
   implicit val formats = org.json4s.DefaultFormats
```

Comment:
Have a look at examples out to extend Default format. You can define NaN -> JNull as an implicit json custom format for this scope.